recipe/meta.yaml

```diff
@@ -19,24 +18,28 @@ requirements:
     - python >=3.6
     - pip
   run:
-    - python >=3.6
-    - robotframework
     - docutils
-    - pyyaml
     - flex-swagger
     - genson
     - jsonpath-ng
-    - pygments
-    - requests
     - jsonschema
-    - rfc3987
-    - strict-rfc3339
+    - pygments
+    - python >=3.6
     - pytz
+    - pyyaml
+    - requests
+    - robotframework
     - tzlocal
 
 test:
+  requires:
+    - pip
+    # TODO: revist after https://github.com/conda-forge/flex-swagger-feedstock/pull/8
+    - python <3.10
```

Comment:
It's not clear to me yet why we need a lower version of Python here?

Is it because the combination of the current flex version and this new restinstance version is not compatible with Python 3.10 and higher? And if so, then merging this might give issues to users, right?

Reply:
@abloemert the upstream PR got merged, so this now works as advertised on 3.10